### PR TITLE
[RFR] Fixed prints

### PR DESCRIPTION
**Pull request by psav**
Fixed prints

### CODE REVIEW

File: cfme/fixtures/rdb.py [hunk 1]
```diff
@@ -103,7 +106,10 @@ def do_continue(self, arg):
     do_c = do_cont = do_continue
 
     def interaction(self, *args, **kwargs):
-        print >>self.stdout, self._prompt_msg
+        if sys.version_info.major == 3:
+            print(self._prompt_msg, stream=self.stdout)
+        else:
+            print >> self.stdout, self._prompt_msg
```
Comment (RonnyPfannschmidt) [inline cfme/fixtures/rdb.py]: **required** remove the python2 syntax

Comment (psav) [reply cfme/fixtures/rdb.py]: Please sugges tan alternative

Comment (RonnyPfannschmidt) [reply cfme/fixtures/rdb.py]: using only the python3 syntax

Comment (psav) [reply cfme/fixtures/rdb.py]: it doesn't work on python 2

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request merged**